website/static/js/myProjects.js

```diff
@@ -812,7 +812,7 @@ var MyProjects = {
                         } else {
                             template = m('.db-non-load-template.m-md.p-md.osf-box',
                             'You have not made any registrations yet. Go to ',
-                            m('a', {href: 'https://openscience.zendesk.com/hc/en-us/categories/360001550953'}, 'Guides'), ' to learn how registrations work.' );
+                            m('a', {href: 'https://help.osf.io'}, 'Guides'), ' to learn how registrations work.' );
```

Comment:
Would https://help.osf.io/article/330-welcome-to-registrations be a better place to link to for `to learn how registrations work`?